packages/teams-js/test/public/media.spec.ts

```diff
@@ -953,153 +963,253 @@ describe('media', () => {
-          expect(message.args.length).toBe(1); // args will be of length 1 for the supported version
+        const message = mobilePlatformMock.findMessageByFunc('getMedia');
+        expect(message).not.toBeNull();
+        expect(message.args.length).toBe(1); // args will be of length 1 for the supported version
 
-          const stringMediaData = 'the media data';
-          const firstMediaResult: media.MediaResult = {
-            error: undefined,
-            mediaChunk: {
-              chunk: btoa(stringMediaData),
-              chunkSequence: 1,
-            },
-          };
-          const secondMediaResult: media.MediaResult = {
-            error: undefined,
-            mediaChunk: {
-              chunk: undefined,
-              chunkSequence: 0,
+        const stringMediaData = 'the media data';
+        const firstMediaResult: media.MediaResult = {
+          error: undefined,
+          mediaChunk: {
+            chunk: btoa(stringMediaData),
+            chunkSequence: 1,
+          },
+        };
+        const secondMediaResult: media.MediaResult = {
+          error: undefined,
+          mediaChunk: {
+            chunk: undefined,
+            chunkSequence: 0,
+          },
+        };
+
+        const mediaResults = Array.of(firstMediaResult, secondMediaResult);
+
+        for (let i = 0; i < mediaResults.length; ++i) {
+          mobilePlatformMock.respondToMessage({
+            data: {
+              id: message.id,
+              args: [mediaResults[i]],
+              isPartialResponse: i < mediaResults.length - 1,
             },
-          };
-
-          const mediaResults = Array.of(firstMediaResult, secondMediaResult);
-
-          for (let i = 0; i < mediaResults.length; ++i) {
-            mobilePlatformMock.respondToMessage({
-              data: {
-                id: message.id,
-                args: [mediaResults[i]],
-                isPartialResponse: i < mediaResults.length - 1,
-              },
-            } as DOMMessageEvent);
-          }
-        });
+          } as DOMMessageEvent);
+        }
       });
 
-      it('getMedia calls with error with MediaCallback', done => {
-        mobilePlatformMock.initializeWithContext(FrameContexts.content).then(() => {
-          mobilePlatformMock.setClientSupportedSDKVersion(getMediaCallbackSupportVersion);
-          const mediaOutput: media.Media = new media.Media();
-          mediaOutput.content = '12345678';
-          mediaOutput.mimeType = 'image/jpeg';
-          mediaOutput.format = media.FileFormat.ID;
-          mediaOutput.getMedia((error: SdkError, blob: Blob) => {
-            expect(error.errorCode).toBe(500);
-            expect(error.message).toEqual('data received is null');
-            expect(blob).toBeFalsy();
-            done();
-          });
+      it('getMedia returns error when called via the callback', async () => {
+        await mobilePlatformMock.initializeWithContext(FrameContexts.content);
```

Comment:
new test, used the USER_ABORT as just a random error code to add to handle the case where there IS a media result, but there is also an error in that object